girder/models/assetstore.py

```diff
@@ -120,7 +120,16 @@ def addComputedInfo(self, assetstore):
         from .file import File
         from girder.utility import assetstore_utilities
 
-        adapter = assetstore_utilities.getAssetstoreAdapter(assetstore)
+        try:
+            adapter = assetstore_utilities.getAssetstoreAdapter(assetstore)
+        except GirderException as e:
+            if e.identifier != assetstore_utilities.EXCEPTION_ID_NO_ADAPTER:
+                raise
+            # If the adapter doesn't exist, use the abstract adapter, since
+            # this will just give the default capacity information
+            from girder.utility.abstract_assetstore_adapter import AbstractAssetstoreAdapter
```

Comment:
Why is it necessary to do the import at runtime here, instead of at import-time of this module?

Reply:
The existing code imported at runtime.  It looks like @zachmullen did that for the File import (is that necessary, since we don't invoke it until called?).  We shouldn't need to do that for the assetstore utilities or the abstract assetstore; I'll change those.